src/main/java/org/isf/visits/gui/InsertVisit.java

```diff
@@ -223,6 +225,37 @@ private JPanel getWardPanel() {
 		return wardPanel;
 	}
 
+	private JComboBox<Ward> getWardBox() {
```

Comment:
Look starting at line 259, the method getWardBox() is already defined in this class.    One of them needs to be deleted to  compile properly.

Reply:
I was sure to have deleted one of the two... thanks!